### DISCUSSION TITLE
Welcome to Ruoyi-Nest-React Discussions!

### DISCUSSION THREAD

**Discussion by PanamaHat123** [PanamaHat123/Ruoyi-Nest-React#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ToDreamr** [comment]
Is this a new vision of ruoyi created by react instead of vue.
